docs/blog/2022-06-17-welcome/index.md

```diff
@@ -0,0 +1,48 @@
+---
+slug: welcome
+title: Welcome
+authors: [maceo]
+tags: [hello, juno]
+---
+
+Juno is a Go implementation of a StarkNet full node client made with ❤️ by Nethermind.
+
+We are working hard for our first release, until that happen, what you can do? 
+
+Let's discover **Juno in less than 5 min**.
+
+## What you'll need
+
+- [Golang](https://go.dev/doc/install) version 1.18 for build and run the project.
+- [Cairo-lang](https://www.cairo-lang.org/docs/quickstart.html) if you want to do `starknet_call` command.
+
+### Installing dependencies
```

Comment:
I'm not sure if it would be good to list dependencies here, users may want to install dependencies on their own or just check easily what they need to run Go. I see the dependencies are `courtney` and `gofumpt`, are these required production dependencies or development dependencies?

Reply:
done